src/main/java/com/ericsson/eiffel/remrem/semantics/eiffelSchemas/CreateSemanticsInputJson.java

```diff
@@ -0,0 +1,54 @@
+package com.ericsson.eiffel.remrem.semantics.eiffelSchemas;
+
+import java.io.File;
+import java.io.FileWriter;
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+
+/**
+ * This class is used to create the new json files in semantics input schema
+ * location which are required to generate the pojo's
+ * 
+ * @author xdurvak
+ *
+ */
+
+public class CreateSemanticsInputJson {
+
+	/**
+	 * This method is used to create json files with corresponding event name
+	 * with json content
+	 * 
+	 * @param jsonFileName
+	 *            - This parameter is used to create the json file with this
+	 *            name
+	 * @param jsonObject
+	 *            - This parameter having the modified json object with all
+	 *            required properties to generate events
+	 */
+	public void createNewJsonSchema(String jsonFileName, JsonObject jsonObject) {
```

Comment:
Creating a class with one method and no instance variables do not improve. Move this method to ChangeJsonSchmeaProperty